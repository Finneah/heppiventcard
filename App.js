import {Root, StyleProvider} from 'native-base';
import React from 'react';

import getTheme from './native-base-theme/components';
import platform from './native-base-theme/variables/platform';
import {NavigationContainer} from '@react-navigation/native';
import HomeScreen from './src/screens/HomeScreen';

import {createStackNavigator} from '@react-navigation/stack';
import {strings} from './src/i18n';

const Stack = createStackNavigator();

const App = () => {
  return (
    <StyleProvider style={getTheme(platform)}>
      <Root>
        <NavigationContainer>
          <Stack.Navigator headerMode="none">
            <Stack.Screen
              name="Home"
              component={HomeScreen}
              options={{
                headerTitle: strings('APPTITLE'),
              }}
            />
          </Stack.Navigator>
        </NavigationContainer>
      </Root>
    </StyleProvider>
  );
};

export default App;
