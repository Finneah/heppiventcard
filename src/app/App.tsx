import {extendTheme, NativeBaseProvider} from 'native-base';
import * as React from 'react';

import {NavigationContainer} from '@react-navigation/native';
import HomeScreen from '../components/HomeScreen/HomeScreen';

import {createStackNavigator} from '@react-navigation/stack';
import {strings} from '../locale/i18n';
import {TourGuideProvider} from 'rn-tourguide';

const Stack = createStackNavigator();

const App: React.FC = () => {
  const theme = extendTheme({
    colors: {
      // Add new color

      primary: {
        50: '#e2f7ff',
        100: '#c3e2ed',
        200: '#a1cddd',
        300: '#7eb9cd',
        400: '#5ca5be',
        500: '#438ca5',
        600: '#326d81',
        700: '#204e5d',
        800: '#0d3039',
        900: '#001217',
      },
      secondary: {
        50: '#ffeedd',
        100: '#ffd1b1',
        200: '#fbb383',
        300: '#f89554',
        400: '#f57724',
        500: '#db5e0a',
        600: '#ab4906',
        700: '#7b3303',
        800: '#4b1e00',
        900: '#1f0700',
      },
    },
    config: {
      // Changing initialColorMode to 'dark'
      initialColorMode: 'light',
    },
  });
  return (
    <TourGuideProvider {...{borderRadius: 16}}>
      <NativeBaseProvider theme={theme}>
        <NavigationContainer>
          <Stack.Navigator headerMode="none">
            <Stack.Screen
              name="Home"
              component={HomeScreen}
              options={{
                headerTitle: strings('APPTITLE'),
              }}
            />
          </Stack.Navigator>
        </NavigationContainer>
      </NativeBaseProvider>
    </TourGuideProvider>
  );
};

export default App;
