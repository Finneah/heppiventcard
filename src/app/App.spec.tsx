/**
 * @format
 */

import 'react-native';
import * as React from 'react';
import App from './App';

// Note: test renderer must be required after react-native.
import {render} from '@testing-library/react-native';

it('renders correctly', () => {
  render(<App />);
});
