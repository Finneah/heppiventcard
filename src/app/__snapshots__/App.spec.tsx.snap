// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />
</View>
`;
